feature/bgp/multipath/otg_tests/bgp_multipath_ecmp_test/metadata.textproto

```diff
@@ -0,0 +1,38 @@
+# proto-file: third_party/openconfig/featureprofiles/proto/metadata.proto
+# proto-message: Metadata
```

Comment:
Should a tag be added to the metadata?
https://github.com/openconfig/featureprofiles/blob/3e6550ab936434cd9ed0b921005e0b2a22323796/proto/metadata.proto#L406-L412